<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>State</class>
 <widget class="QDialog" name="State">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>96</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>State</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QGroupBox" name="state">
     <property name="title">
      <string>State: state</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="3">
       <widget class="QLabel" name="label_step_max">
        <property name="text">
         <string>00000</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="text_value_min">
        <property name="text">
         <string>Min:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="label_element_min">
        <property name="text">
         <string>00000</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="label_element_max">
        <property name="text">
         <string>00000</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="text_node_min">
        <property name="text">
         <string>Node:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="text_element_max">
        <property name="text">
         <string>Element:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="text_element_min">
        <property name="text">
         <string>Element:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QLabel" name="text_node_max">
        <property name="text">
         <string>Node:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_step_min">
        <property name="text">
         <string>00000</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="text_step_max">
        <property name="text">
         <string>Step:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="text_value_max">
        <property name="text">
         <string>Max:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_value_min">
        <property name="text">
         <string>+0.00e+00</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_value_max">
        <property name="text">
         <string>+0.00e+00</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="text_step_min">
        <property name="text">
         <string>Step:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QLabel" name="label_node_min">
        <property name="text">
         <string>00 (00000)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QLabel" name="label_node_max">
        <property name="text">
         <string>00 (00000)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="solver">
     <property name="title">
      <string>Solver</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="text_solver_min">
        <property name="text">
         <string>Min:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="text_solver_max">
        <property name="text">
         <string>Max:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_solver_min">
        <property name="text">
         <string>+0.00e+00</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_solver_max">
        <property name="text">
         <string>+0.00e+00</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="Line" name="separator">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
